### DISCUSSION TITLE
How to remove certificates permanently not only the symlink

### DISCUSSION THREAD

**Discussion by pupattan** [nginx-proxy/acme-companion#850] [Q&A]
How to remove certificates permanently not only the symlink ?

**Comment by buchdag** [comment]
You'll have to remove them from both the `/etc/acme.sh` and `/etc/nginx/certs`

Let say the domain you want to delete is `subdomain.example.com` and the email address you used is `contact@example.com`.

The certificates will be persisted in those two folders:
- `/etc/acme.sh/contact@example.com/subdomain.example.com`
- `/etc/nginx/certs/subdomain.example.com`

If your certificate is a multi domain (SAN) certificate, the name of the folders will be the first domain on the `LETSENCRYPT_HOST` environment variable when you issued the certificate.

If you want to delete a certificate that was issued for `LETSENCRYPT_HOST=foo.example.com,bar.example.com,baz.example.com`, the name of the actual folders containing the certificate will be `foo.example.com`.

**Reply by pupattan** [reply]
Thanks
